daprdocs/content/en/operations/resiliency/configure-policies.md

```diff
@@ -0,0 +1,282 @@
+---
+type: docs
+title: "How-To: Error recovery using resiliency policies"
+linkTitle: "Resiliency Policies"
+weight: 4500
+description: "Configure Dapr error retries, timeouts and circuit breakers"
+---
+
+Resiliency is currently a preview feature. Before you can utilize resiliency policies you must first [enable the resiliency preview feature]({{<ref preview-features >}}).
+
+## Introduction
+
+- TODO: What is resiliency in Dapr?
+- TODO: What problems does it solve?
+
+## Overview
+
+A Dapr resiliency policy allow retry, timeout and circuitbreaker policies to be created and applied to particular targets including specific components and apps (service invocation calls to other apps).
+
+Additionally, resiliency policies can also be [scoped to specific apps]({{<ref "component-scopes.md#application-access-to-components-with-scopes">}}).
```

Comment:
@berndverst we need a section here to talk about service invocation specifically. The reason is that service invocation already has retries built-in. For every failed call, Dapr automatically retries up to 3 times: https://docs.dapr.io/developing-applications/building-blocks/service-invocation/service-invocation-overview/#retries